example.config.py

```diff
@@ -1,6 +1,7 @@
-token = "YOUR_DISCORD_BOT_TOKEN_HERE" # Obviously you need to specify this.
-groq_api_key = "YOUR_GROQ_OR_OPENAI_API_KEY" # Will disable the AI functions if not provided
+token = "BOT_TOKEN"
+ai_api_key = "GROQ_API_KEY"
```

Comment:
Name this something that makes it apparent that both a groq and openai key can be entered